probe-rs/src/probe/glasgow/usb.rs

```diff
@@ -21,7 +21,10 @@ pub struct GlasgowUsbDevice {
 
 impl GlasgowUsbDevice {
     pub fn new_from_selector(selector: &DebugProbeSelector) -> Result<Self, ProbeCreationError> {
-        let Some(serial) = selector.serial_number.clone() else {
+        if selector.vendor_id != 0x20b7 && selector.product_id != 0x9db1 {
```

Comment:
I would like if those magic VID/PID would have proper const names :)

Reply:
I would like @whitequark to tell me whether this is okay in the first place :D

Reply:
Yeah, seems fine.

```suggestion
        if selector.vendor_id != VID_QIHW && selector.product_id != PID_GLASGOW {
```